tests/deprecated_flag_json_schema_test.fbs

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
there is already a deprecated field in the main `monster_test` schema, lets not add further schemas just for this feature

Reply:
removed